indy-besu/smart_contracts/contracts/did/DidErrors.sol

```diff
@@ -26,6 +26,11 @@ error DidAlreadyExist(string did);
  */
 error DidHasBeenDeactivated(string did);
 
+/**
```

Comment:
Please update design documentation to reflect changes in the state object and valdiation

Reply:
It already has requirement `Required role` = `DID Owner`